site/static/compare.html

```diff
@@ -274,53 +274,21 @@ <h2>Comparing <span id="stat-header">{{stat}}</span> between <span id="before">{
                     </div>
                     <input type="checkbox" v-model="showRawData" style="margin-left: 20px;" />
                 </div>
+                <button @click="resetFilter">Reset filters</button>
             </div>
         </fieldset>
         <p v-if="dataLoading && !data">Loading ...</p>
         <div v-if="data" id="content" style="margin-top: 15px">
-            <div id="summary">
-                <div style="display: flex; justify-content: end;">
+            <div id="main-summary">
+                <summary-table :cases="filteredSummary"></summary-table>
+                <div style="position: absolute; right: 5px; top: 5px;">
                     <span class="tooltip" style="margin-right: 1em;">?
                         <span class="tooltiptext">
-                            The percents show arithmetic mean amongst regressions, amongst improvements
-                            and finally amongst all benchmarks in each category (all benchmarks or
-                            filtered benchmarks).
+                            The table shows summaries of regressions, improvements and all changes
```

Comment:
Perhaps we can make clear that this is all the *visible* data?

Reply:
Changed the help content to make this more clear (hopefully :) ).